docs/production.md

```diff
@@ -0,0 +1,69 @@
+## From development to production
+
+After releasing your app to the world, we recommend creating a development branch for working on new features and bug fixes.
+
+Using a development branch, you can safely experiment with schema changes while minimising the risk of data loss, downtime, or compatibility issues between your app and production database.
+
+### Create a development branch
+
+Simply ask the LLM to "create a development branch", and it will invoke the `create_branch` MCP tool.
+
+The development branch clones your production branch by applying the same migrations shown by `list_migrations` tool. It does not include any untracked data or schema changes that came directly from users interacting with your app.
+
+Depending on the size of your migrations, your development branch may take up to a few minutes to setup. You can ask the LLM to check the branch status periodically using the `list_branches` tool. 
+
+### Create a new migration
+
+Once your development is ready, you can start building new features by invoking the `apply_migration` tool. This tool tracks any schema or data changes as a migration so that it can be replayed on your production branch when you are ready to deploy.
```

Comment:
```suggestion
Once your development branch is ready, you can start building new features by invoking the `apply_migration` tool. This tool tracks any schema or data changes as a migration so that it can be replayed on your production branch when you are ready to deploy.
```